sdk/src/types/prekey.rs

```diff
@@ -0,0 +1,239 @@
+use {
```

Comment:
this file should be unnecessary imo

Reply:
worst case you can create a temporary struct in `auth.rs` like so:

```
struct RawPreKey {
  bytes: Vec<u8>
}
```

and fetch this via the object crawler.

then just `bincode::deserialize::<PreKeyBundle>(&raw)`